Fluent.Ribbon/Themes/Controls/BackstageTabControl.xaml

```diff
@@ -58,7 +58,7 @@
            TargetType="{x:Type Fluent:Button}">
         <Setter Property="Fluent:RibbonProperties.IconSize" Value="Small" />
         <Setter Property="Fluent:RibbonProperties.MouseOverBackground" Value="{DynamicResource Fluent.Ribbon.Brushes.BackstageTabItem.MouseOver.Background}" />
-        <Setter Property="FocusVisualStyle" Value="{DynamicResource Fluent.Ribbon.Styles.ControlStyleEmptyFocus}" />
+        <Setter Property="FocusVisualStyle" Value="{StaticResource Fluent.Ribbon.Styles.FocusVisual.Tight}" />
```

Comment:
All resource usages have to use DynamicResource.
Otherwise people won't be able to overwrite those.